client_code/messaging.py

```diff
@@ -26,11 +29,10 @@ def __init__(self, subscriber, handler):
 
 
 class Publisher:
-    default_logger = None
     default_log_level = INFO
 
     def __init__(self, logger: _Logger = None, **kwargs):
```

Comment:
```suggestion
    def __init__(self, *, logger: _Logger = None, **kwargs):
```

Make logger keyword only (not completed relevant to this PR but while we're here)